msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,322 @@
+* The iOS or Android version of the app must not show any UI or language or link to any other apps, add-ins, or websites that ask the user to pay.
+* The associated privacy policy and terms of use pages must also be free of any commerce UI or store links.
+* If your app contains tabs, they must follow the mobile design guidelines.
+* App should function on the following platforms:
+    * iOS 9.0 and later
+    * Android 4.4 and later
+    * Windows Phone 10.0.10586 and later
+
+## 4.0 App package and store listing
+
+App packages must be correctly formatted and include all required information and components.
+
+### 4.1 App manifest
+
+* The app manifest must conform to the [latest manifest schema](~/resources/schema/manifest-schema.md).
+* If your app includes a bot or messaging extension, the manifest must be consistent with Bot Framework metadata, including bot name, logo, privacy policy link, and terms of service link.
+* If your app uses Azure Active Directory (AD) for authentication, include the Azure AD ID in the manifest ([see how](~/resources/schema/manifest-schema.md#webapplicationinfo)).
+
+### 4.2 App icons
+
+* Your app package must include two PNG versions of your app icon: a color icon and an outline icon.
+* A correctly sized and formatted icons must be specified in the app package or manifest. The icons in the package or manifest must match the icon submitted with the offer metadata.
+* For detailed information, refer to the [icons documentation](~/concepts/build-and-test/apps-package.md#app-icons).
+
+### 4.3 App description
+
+The app descriptions in your manifest and store should be same and follow these guidelines:
+
+* Use "Microsoft Teams" on first reference. Subsequent references can be shortened to ”Teams”.
+* Do not abbreviate "Microsoft" (such as "MS" or "MSFT").
+* Avoid unnecessary capitalization and language errors.
+* Follow the [short description guidelines](~/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md#short-description).
+* Follow the [long description guidelines](~/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md#long-description).
+* Clearly describe any limitations, conditions, or exceptions to the functionality, features, and deliverables described in the listing and related materials before the customer acquires your offer. The capabilities you declare must relate to the core functions and description of your offer.
+
+### 4.4 Localization
+
+Your package must include a localization file that includes language translations that display based on the Teams language settings. The file must conform to the [Teams localization schema](~/concepts/build-and-test/apps-localization.md).
+
+## 5.0 Tab design
+
+See comprehensive [tab design guidelines](~/tabs/design/tabs.md).
+
+### 5.1 Setup
+
+Tab set up must not dead-end the new user. A way forward message should be provided to complete the workflow.
```

Comment:
Should the second sentence include “must” and not "should" to be consistent with the first sentence?

Reply:
Yes. Let's change it to must